conf/application.conf

```diff
@@ -92,6 +92,8 @@ features {
   # if enableFrontpage is true, / will always be the /spotlight page
   # if it is false, / will be the login page, if not logged in, and the dashboard otherwise
   enableFrontpage = false
+  # The default organization used in the onboard flow (wrapped in "). If you don't want one use null or "".
+  defaultOnboardingOrganization = null
```

Comment:
There already is a `defaultOrganization` within the application.conf. Please use this attribute instead. Unless there is a specific reason against it?

Reply:
The reason I did not use it in the first iteration is, that there might be the case where you want the default organization to be different in onboarding and in the usual registration (afaik the current application of `defaultOrganization`).
If you think they both are related and should be always the same, I will change it :slightly_smiling_face: I was just unsure about this :)

Reply:
The relevant use case for us is the demo deployment for which we always want to default to the demo org. I cannot think of a scenario, in which you'd want two different defaults depending on which flow you used :)